jquery.confirm.js

```diff
@@ -101,6 +101,13 @@
                     '<h4 class="modal-title">' + settings.title+'</h4>' +
                 '</div>';
         }
+        var cancelButtonHtml = '';
+        if (settings.cancelButton) {
```

Comment:
Oh wait you are re-using the "cancel button" label option :) How about updating the documentation of that option in the README: https://github.com/myclabs/jquery.confirm#options ?